### PR TITLE
BWS-167: + distance-from-the-field-to-validation-error-message

### PR DESCRIPTION
**Pull request by ania-leichenko**
Screenshot_89

### CODE REVIEW

**Review by what1s1ove — CHANGES_REQUESTED** [review]
we should add gap between last input and the submit button
you can use `gap` for the form element

File: frontend/src/components/common/input/input.module.scss [hunk 1]
```diff
@@ -42,6 +42,8 @@
 }
 
 .error {
+  margin-top: 8px;
+  margin-bottom: 12px;
```
Comment (what1s1ove) [inline frontend/src/components/common/input/input.module.scss]:
```suggestion
.input {
  margin: 0 0 5px;
```
common component should know anything about spaces between them. add spaces on the `form` level

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/input/input.module.scss (hunk 1 above)
Comment (what1s1ove) [reply frontend/src/components/common/input/input.module.scss]: don't forget about textaria

Comment (Yaroslaww-1) [reply frontend/src/components/common/input/input.module.scss]: I think distance between input and its error message is too large. We need to reduce this distance

**Review by junojei — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**